boards/arduino-atmega-common/include/gpio_params.h

```diff
@@ -34,7 +34,8 @@ static const  saul_gpio_params_t saul_gpio_params[] =
     {
         .name = "LED",
         .pin = LED0_PIN,
-        .mode = GPIO_OUT
+        .mode = GPIO_OUT,
+        .flags = 0,
```

Comment:
(here and below) when using designated initializers, empty fields don't have to be specified. So IMO the ```.flags = 0``` should go.

Reply:
That what I was thinking too, but when talking with @haukepetersen he thought they may be required for some architecture.
I will remove them all.